src/scenes/gameOver.js

```diff
@@ -0,0 +1,62 @@
+export default class GameOver extends Phaser.Scene {
+  constructor() {
+    super({ key: "gameOver" });
+  }
+
+  init(data) {
+    this.name = data.name;
+    this.number = data.number;
+    this.next = data.next;
+  }
+
+  create() {
+    this.add.image(0, 0, "menuBackground").setOrigin(0, 0);
+    this.width = this.sys.game.config.width;
+    this.height = this.sys.game.config.height;
+    this.centerWidth = this.width / 2;
+    this.centerHeight = this.height / 2;
+    this.introLayer = this.add.layer();
+
+    this.text = [
+      "GAME OVER",
+      "You fought bravely, but the horde was too strong.",
+      "Your journey ends here.",
+      "Press SPACE to restart",
+    ];
+
+    this.showMessages();
+
+    this.input.keyboard.on("keydown-SPACE", () => this.restartGame(), this);
```

Comment:
**praise**: Nice touch adding both SPACE and ENTER keys for restarting the game, enhancing user experience.